src/components/Calculator.js

```diff
@@ -0,0 +1,21 @@
+export default function Calculator() {
+  const gumbs = ['AC', '+/-', '%', '/', 7, 8, 9, 'x', 4, 5, 6, '-', 1, 2, 3, '+', 0, '.', '='];
+
+  return (
+    <section id="calculator">
+      <input
+        type="text"
+        value="0"
+        disabled
+        id="exit"
+      />
+      <section id="input">
+        <article id="digits">
+          {
+            gumbs.map((gumb) => <button key={gumb} type="button">{gumb}</button>)
+          }
+        </article>
+      </section>
+    </section>
```

Comment:
- [ ] Please kindly work on all the background-colors of your app so as to match the given template. I noticed it was specifically stated in the project requirement that your app should match the given design. When we give attention to details, it will really enhance our software development journey. 👍 

## Your Design
![Untitled](https://user-images.githubusercontent.com/103386183/220347460-fb947ea7-e707-40d4-b02e-61c00e744368.jpg)

## Given
![Untitled](https://user-images.githubusercontent.com/103386183/220347603-79ca01e7-3f2e-4f09-8f63-3812058f77ee.jpg)
 Design